spider-foundational-work/utility-scripts/domains.txt

```diff
@@ -0,0 +1,66 @@
+armymwr.com
```

Comment:
I'm not entirely sure the specifics but yeah, it would replace the calls to whatever the Medusa gem implementation is. We should definitely have a meeting with Jim/Dawn down the road to hammer out the specifics.

Reply:
ultimately, yes, we would want to be managing the crawling domains in superadmin